lib/galaxy/web/framework/base.py

```diff
@@ -344,7 +339,7 @@ def _read_lines(self):
     # Adapt `self.__file = None` to Python name mangling of class-private attributes.
     # We need to patch the original FieldStorage class attribute, not the cgi_FieldStorage
     # class.
-    setattr(self, '_FieldStorage__file', None)
+    self._FieldStorage__file = None
```

Comment:
I don't know if that is completely equivalent, seems a bit risky. This needs careful checking in the regular uploader and the history archive uploader. I remember this was a pain when I set this up.

Reply:
According to the documentation they are equivalent: https://docs.python.org/3/library/functions.html#setattr
Using `setattr` become unnecessary at commit https://github.com/galaxyproject/galaxy/commit/1dc4cf224108db9756a96b9d50a71f9f099a00f2 , when the attribute name was changed to a fixed string.

Reply:
Awesome, thanks for clarifying that!